lib/services/service.dart

```diff
@@ -8,7 +8,7 @@ class ReturnHttpPost<T> {
 }
 
 class HttpService {
-  final String address = "http://127.0.0.1/api";
+  static final String address = "http://192.168.1.104/api";
```

Comment:
There will be `[SocketException](https://stackoverflow.com/questions/55785581/socketexception-os-error-connection-refused-errno-111-in-flutter-using-djan)` when using `127.0.0.1`, so I use my local ipv4 address.